### PR TITLE
Update instructions to create Bitbucket user key

### LINKED ISSUE #4830: Instructions (steps and screenshots) to create a Bitbucket user key pertain to the old/legacy UI
# Missing Information
How to create Bitbucket user keys in the new UI. (The current instructions apply to the legacy UI.)

# Already Looked
https://circleci.com/docs/2.0/gh-bb-integration/#creating-a-bitbucket-user-key

# Potential Locations
Not located anywhere else. Documentation needs to be updated to be consistent with the new UI.

### PR DESCRIPTION
**Pull request by michizhou**
# Description
Updated instructions on how to create a Bitbucket user key to be more consistent with the new UI for CircleCI, replacing the current instructions that applied to the legacy UI for CircleCI.

# Reasons
Fixes #4830

### CODE REVIEW

**Comment by rosieyohannan** [comment]
Thank you so much for bringing this up to date, @michizhou! ☺️

File: jekyll/_cci2/gh-bb-integration.md [hunk 1]
```diff
@@ -296,20 +296,24 @@ by following this workaround:
 1. In the CircleCI application,
 go to your project's settings.
 
-2. Navigate to the **Checkout SSH Keys** page.
+2. Navigate to the **SSH Keys** page.
 
-3. Right-click the **Create `<username>` user key** button
-and select the **Inspect** option to open the browser inspector.![]( {{ site.baseurl }}/assets/img/docs/bb_user_key.png)
+3. - **For users of the new CircleCI UI:** Right-click the **Add User Key** button and select the **Inspect** option to open the browser inspector.![]( {{ site.baseurl }}/assets/img/docs/new_bb_user_key.png)
```
Comment (rosieyohannan) [inline jekyll/_cci2/gh-bb-integration.md:301]: Would you mind just removing references to using the new UI? Now that everyone is using it we can just go with the steps as they are - in this case just remove: "**For users of the new CircleCI UI:**"

**Comment by michizhou** [comment]
@rosieyohannan I have just updated the documentation to remove all references to the new CircleCI UI. Please let me know if any additional changes need to be made. Thank you.

File: jekyll/_cci2/gh-bb-integration.md [hunk 2]
```diff
@@ -296,20 +296,24 @@ by following this workaround:
 1. In the CircleCI application,
 go to your project's settings.
 
-2. Navigate to the **Checkout SSH Keys** page.
+2. Navigate to the **SSH Keys** page.
 
-3. Right-click the **Create `<username>` user key** button
-and select the **Inspect** option to open the browser inspector.![]( {{ site.baseurl }}/assets/img/docs/bb_user_key.png)
+3. Right-click the **Add User Key** button and select the **Inspect** option to open the browser inspector.![]( {{ site.baseurl }}/assets/img/docs/new_bb_user_key.png)
+    - **For users of the legacy CircleCI UI:** Right-click the **Create `<username>` user key** button and select the **Inspect** option to open the browser inspector.![]( {{ site.baseurl }}/assets/img/docs/bb_user_key.png)
```
Comment (rosieyohannan) [inline jekyll/_cci2/gh-bb-integration.md:302]: Sorry, should have said the legacy UI is only used by CircleCI server customers now, and Bitbucket isn't supported in that product line, so these references can be removed too :-)

**Comment by michizhou** [comment]
@rosieyohannan I deeply apologize for being silent on this PR for the past few months. I just merged with the existing master code via my latest commit, which addresses your last comments. If you need anything else from me, please let me know.

**Comment by yaningo** [comment]
@thomasr8 This PR was closed rather than merged, so the current documentation is still outdated.

**Pull request closed without merging**